packages/css/src/field.css

```diff
@@ -101,7 +101,8 @@
 
   & .ds-field-affix {
     align-items: center;
-    border: var(--dsc-field-affix-border);
+    border: 1px solid;
```

Comment:
Skal jeg ikke få endre på border-bredden?

Reply:
> border: now uses two properties. The color is a variable, and the width and style is a css prop.

Reply:
Og, for å resolve denne, sia denne bruka `border: 1px solid` fleire plassa, så har den no
```css
  --dsc-field-affix-border: 1px solid;
  --dsc-field-affix-border-color: var(--ds-color-neutral-border-default);
```

Eg oppdaterte også beskrivelsen på PRen